src/api/models/Activity.ts

```diff
@@ -25,12 +25,16 @@ import {
 import sequelize from '../../config/sequelize';
 import { Genre } from './Genre';
 import { Label } from './Label';
+import {Address} from "./Address";
 import { ActivityReview } from './ActivityReview';
 
 class Activity extends Model<InferAttributes<Activity, { omit: 'activityReviews' }>, InferCreationAttributes<Activity, { omit: 'activityReviews' }>> {
     declare id: CreationOptional<number>;
     declare title: string;
     declare description: string;
+    declare costPerIndividual: number | null;
+    declare costPerGroup: number | null;
+    declare image: string | null;
     declare startTime: Date | null;
     declare endTime: Date | null;
```

Comment:
Do we have to defined the value has null in the request object? Or they can be without values (undefined)?
This will have them accept undefined:
```suggestion
    declare costPerIndividual?: number;
    declare costPerGroup?: number;
    declare image?: string;
    declare startTime?: Date;
    declare endTime?: Date;
```

Reply:
My thought is it's better if they have a default value rather than null (except the endTime, I didn't think it through enough lol)